### PR TITLE
[fix](Statistics)Drop stats or update updated rows after truncate table.

### PR DESCRIPTION
**Pull request by Jibing-Li**
1. Also clear follower's stats cache when doing drop stats.
2. Drop stats when truncate a table.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by Kikyou1997 — APPROVED** [review]

**Comment by Jibing-Li** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by Jibing-Li** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java [hunk 1]
```diff
@@ -3065,7 +3068,13 @@ public void truncateTable(TruncateTableStmt truncateTableStmt) throws DdlExcepti
         } finally {
             olapTable.writeUnlock();
         }
-
+        if (truncateEntireTable) {
+            // Drop the whole table stats after truncate the entire table
+            Env.getCurrentEnv().getAnalysisManager().dropStats(olapTable);
```
[the remark this answers is not in the document]
Comment (Jibing-Li) [reply fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java:3073]: Yes. Changed drop stats logic, will clear followers' cache while dropping stats.

**Comment by Jibing-Li** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by Kikyou1997 — APPROVED** [review]

**Comment by morrySnow** [comment]
run buildall

**Pull request merged**